README.md

```diff
@@ -1,2 +1,42 @@
 # crispy-api
 Typescript GraphQL API for `crispy-web`
+
+API based on [`Apollo`](https://www.apollographql.com/docs/apollo-server/) express server. 
+
+## How to use
+
+### 1. Download project & install dependencies
+
+Clone the `crispy-api` branch of this repository:
+
+```
+git clone git@github.com:WiseEngineering/crispy-api.git
+```
+
+Go to project folder:
+
+```
+cd crispy-api
+cp .env.example .env
+```
+
+
+### 2. Start up `Docker` servers 
+
+Command to start up server
+
+```
+docker-compose up -d
+```
+
+Run migration:
+
+```
+docker exec -it <container name> yarn migration
```

Comment:
maybe worth to specify names in docker-compose to be able  put command here like
```
docker exec -it  crispy-api yarn migrate
```